packages/starknet_provider/test/get_block_with_receipts_test.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:test/test.dart';
+import 'package:starknet_provider/starknet_provider.dart';
+import 'package:starknet_provider/src/model/block_with_receipts.dart';
```

Comment:
This import should not be needed.
I think you forgot to export your model in model/index.dart